src/App.js

```diff
@@ -1,21 +1,21 @@
 import './App.css';
 import React, { Component } from 'react'
 import Find from './Find'
-import CardFront from './CardFront'
+import favoritePlayer from './SinglePlayerData.json'
+import CardBack from './CardBack'
 import { getAllCubsPlayers, online } from './Utility';
 import { Route, Switch } from 'react-router-dom';
-import favoritePlayer from './SinglePlayerData.json'
-
+import CardFront from './CardFront';
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
       allPlayers: [],
-      activePlayers: []
+      activePlayers: [],
+      chosenPlayer: favoritePlayer,
+      showCardSide: 'front'
```

Comment:
Added this variable so that the app could tell which side of the card should be showing. It's kind of working? Except that when you choose a new player while viewing the back of a card, it takes you to the back of the new player's card. This ... might be a feature and not a bug.